cellfinder/core/classify/classify.py

```diff
@@ -48,9 +48,7 @@ def main(
         callbacks = None
 
     # Too many workers doesn't increase speed, and uses huge amounts of RAM
-    workers = get_num_processes(
-        min_free_cpu_cores=n_free_cpus, n_max_processes=max_workers
-    )
+    workers = get_num_processes(min_free_cpu_cores=n_free_cpus)
```

Comment:
From what I could tell `max_workers` wasn't passed when the this function was called in `core.py` this meant the default value of 3 was used drastically limiting the number of cores. 

We can either remove `max_workers` completely, set the default value to be large (>64?) or expose it in the brainmapper CLI and the napari plugin.

Reply:
This was originally in there because using lots of workers:
- Used huge amounts of RAM
- Didn't speed things up much (presumably the bottleneck was elsewhere)

Do you know if this is still true?

Reply:
I usually used 31 workers on a 32 core CPU when testing the accuracy/performance and didn't notice any excess RAM usage. I'm not sure about the performance, it would be good to profile different number of workers across a small and a large dataset.